### ISSUE TITLE
Accessibility Updates

### ISSUE THREAD

**Issue by brittanyrw** [brittanyrw/emojipages#37]
Make sure that EmojiPages is fully accessible. 

- [ ] Run the site through accessibility tools such as [AXE](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) or [Accessibility Insights for the Web](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
- [ ] Go through [this accessibility checklist](https://a11yproject.com/checklist/)

Either make accessibility updates yourself or create Issues for things that need to be updated.

**Comment by david-webber** [comment]
I'm happy to do this.

**Comment by david-webber** [comment]
I ran both AXE and Accessibility Insights on emojipages, the only issue was a missing lang attribute on the html element. I will open a PR for this.

Do you have any ideas for handling alt text on the emojis?

**Comment by david-webber** [comment]
Setting the font-size to 200% (one of the things in the accessibility checklist) causes the grid to go wider than the viewport.

**Comment by brittanyrw** [comment]
@david-webber We don't have to add alt text on the emojis, it is built in. For example, if you use a voice over tool to read out what is on a webpage, it will read the official name of the emoji. So for example 😂this one is "Face With Tears of Joy" and this one is 🤣"Rolling on the Floor Laughing".

Okay, I will try to merge that other Issue in soon then we can see about the font sizing.

Another accessibility item that can be updated is the focus outline that shows up when you navigate the site using tab. Right now it is the default blue color. Examples below around the button and cards:

<img width="270" alt="Screen Shot 2019-10-01 at 7 02 43 PM" src="https://user-images.githubusercontent.com/23530405/66006841-1cb2c180-e47e-11e9-8a5a-d928db6fe898.png">
<img width="377" alt="Screen Shot 2019-10-01 at 7 02 30 PM" src="https://user-images.githubusercontent.com/23530405/66006843-1cb2c180-e47e-11e9-8898-b37e614477de.png">

Since the background is so dark, it is a little hard to see the focus. We can change the color so that it is more prominent and also make it so that the focus isn't touching the edge of what it is surrounding so that it is easier to see what we are focusing on.

So like this for example if the focus was white and had an offset:
<img width="246" alt="Screen Shot 2019-10-01 at 7 06 25 PM" src="https://user-images.githubusercontent.com/23530405/66006967-98ad0980-e47e-11e9-863c-f571dd3d72b6.png">
<img width="334" alt="Screen Shot 2019-10-01 at 7 06 13 PM" src="https://user-images.githubusercontent.com/23530405/66006968-98ad0980-e47e-11e9-90ac-0989e866da25.png">

But the focus for the hint icon will need to be a different color or remain the default blue because this is what it looks like white:
<img width="96" alt="Screen Shot 2019-10-01 at 7 06 19 PM" src="https://user-images.githubusercontent.com/23530405/66007013-b0848d80-e47e-11e9-9990-b70c9850387f.png">

Let me know if you'd like to tackle that one.

**Comment by david-webber** [comment]
Oh I had no idea emojis we're accessible by default, that's cool! Yeah I'll take a look at the outline issue tomorrow or the next day and then I'll do the pull request.

Thanks.

**Comment by david-webber** [comment]
Hi @brittanyrw - I see you have now opened this up again. I made some changes for this last week before you closed it,

I have modified the focus state for the contribute-callout button and the emoji-images class.
I have changed the outline to a 1px solid white line. I am unable to use the default outline 'blur' style when mixing with an outline-offset (I can change the colour to white but it will ignore the offset, I am unable to reproduce your example above unfortunately, unless you are aware of a way to set an offset whilst retaining the blurred style?).

I have added a slight box-shadow to the focus state on the emoji-images class and :after element.

Please see preview of my changes, if you would like me to make any other changes let me know, if not I will open a PR.

![focus1](https://user-images.githubusercontent.com/15183620/66480189-f6041480-ea95-11e9-9d9f-aea18831572c.JPG)
![focus2](https://user-images.githubusercontent.com/15183620/66480190-f6041480-ea95-11e9-85a9-f18cadd77d47.JPG)
![focus3](https://user-images.githubusercontent.com/15183620/66480191-f69cab00-ea95-11e9-862e-15f81c235885.JPG)



Thanks.
